skeleton/src/main/java/com/dropbox/kaiken/skeleton/dagger/SdkSpec.kt

```diff
@@ -13,3 +18,5 @@ import com.squareup.anvil.annotations.ContributesTo
 interface SdkSpec {
     fun getSkeletonConfig(): SkeletonConfig
 }
+
+interface SkeletonComponent : SdkSpec
```

Comment:
I initially had tried to add:
`@MergeComponent(SkeletonScope::class)
@SingleIn(SkeletonScope::class)
interface RealAppComponent : SkeletonComponent {
    @Component.Factory
    interface Factory {
        fun create(
            @BindsInstance app: SkeletonOwnerApplication,
        ): RealAppComponent
    }
}`

But ran into ` error: [Dagger/MissingBinding] com.dropbox.kaiken.skeleton.core.SkeletonConfig cannot be provided without an @Provides-annotated method.` which didn't quite make sense to me. Open to suggestions if we can improve this!